### PR TITLE
[Reviewer: Rob] Config changes

### PR DESCRIPTION
**Pull request by eleanor-merry**
Homestead changes to match https://github.com/Metaswitch/sprout/pull/1364

### CODE REVIEW

File: debian/homestead.init.d [hunk 1]
```diff
@@ -122,8 +122,11 @@ get_settings()
         if [ -n "$scscf_uri" ]
         then
           server_name=$scscf_uri
+        elif [ -n "$scscf_prefix" ]
+        then
+          server_name="sip:$scscf_prefix.$sprout_hostname:$scscf;transport=TCP"
         else
-          server_name="sip:$(python /usr/share/clearwater/bin/bracket_ipv6_address.py $sprout_hostname):$scscf;transport=TCP"
```
Comment (rkd-msw) [inline debian/homestead.init.d]: Is this right? Aren't you going to end up with "scscf.10.0.0.3"?

e.g. on a non-AWS AIO node: https://github.com/Metaswitch/clearwater-infrastructure/blob/master/debian/clearwater-auto-config-generic.init.d#L72

**Comment by rkd-msw** [comment]
One comment.

File: debian/homestead.init.d (hunk 1 above)
Comment (rkd-msw) [inline debian/homestead.init.d]: We discussed - this is part of the https://github.com/Metaswitch/sprout/issues/929 weirdness but should 'just work'.

Comment (eleanor-merry) [inline debian/homestead.init.d]: It's not that bug - it's just when sprout_hostname isn't a hostname. It should still 'just work'

**Pull request merged**